docs/resources/control-flow/user-interactivity/forms/form-widgets/checkbox.md

```diff
@@ -0,0 +1,527 @@
+---
+slug: /resources/forms/checkbox
+title: Checkboxes
+tags: [Widgets, Form Elements]
+keywords: [FlutterFlow, Checkbox, CheckboxGroup, CheckboxListTile, Multiple Choice]
+description: Learn how to add Checkbox, CheckboxGroup, and CheckboxListTile widget in your FlutterFlow app.
+sidebar_position: 3
+---
+# Checkboxes
```

Comment:
Just Checkbox

Reply:
To differentiate just say Checkbox Widgets here on the title and then Checkbox later as h2

Reply:
Done!